config/unicorn.rb

```diff
@@ -1,11 +1,12 @@
-app_path = File.expand_path('../../', __FILE__)
+app_path = File.expand_path('../../../', __FILE__)
```

Comment:
> Style/ExpandPathArguments: Use `expand_path('../..', __dir__)` instead of `expand_path('../../../', __FILE__)`.
> [link](https://sideci.com/gh/128728288/pull_requests/5)

Capistranoインストールの際記述変更